faiss/utils/hamming.h

```diff
@@ -135,7 +135,8 @@ void hammings_knn_hc(
         size_t nb,
         size_t ncodes,
         int ordered,
-        ApproxTopK_mode_t approx_topk_mode = ApproxTopK_mode_t::EXACT_TOPK);
+        ApproxTopK_mode_t approx_topk_mode = ApproxTopK_mode_t::EXACT_TOPK,
+        const IDSelector* sel = nullptr);
```

Comment:
`#include <faiss/impl/IDSelector.h>` is required at the top of the file as well.